prow/jobs/kyma-project/kyma/tests/components/application-connector/central-application-connectivity-validator/tests-validator.yaml

```diff
@@ -0,0 +1,113 @@
+presubmits:
+  kyma-project/kyma:
+    - name: pre-main-kyma-integration-k3d-app-conn-validator
+      annotations:
+        description: "It tests the acceptance criteria of the (Central) Application Connectivity Validator component"
+        owner: "framefrog"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-integration-k3d-app-conn-validator"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-gc-project-env: "true"
+        preset-kyma-guard-bot-github-token: "true"
+        preset-kyma-integration-app-connector-tests-app-validator: "true"
+        preset-kyma-integration-central-app-connectivity-enabled: "true"
```

Comment:
Do you rely on those presets in `k3d-validator-tests` make target?

Reply:
Neither target depends on these preps, I will remove them from any Prow Job.